pysap/base/transform.py

```diff
@@ -90,6 +90,7 @@ def __init__(self, nb_scale, verbose=0, **kwargs):
         self.scales_lengths = None
         self.scales_padds = None
         self.use_wrapping = pysparse is None
+        self.data_dim = dim
```

Comment:
Do we really need to define this parameter.
For instance, can we guess it from the tranformation name?